### PR TITLE
Add Speedy Latest

### PR DESCRIPTION
**Pull request by clettenberg**

### CODE REVIEW

File: Casks/speedy.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'speedy' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.apimac.com/download/Speedy.zip'
+  name 'Speedy'
+  homepage 'http://www.apimac.com/mac/speedy/'
+  license :commercial    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/speedy.rb]: Please remove the comment.

**Comment by fanquake** [comment]
@cacqw7 Can you please squash your commits?

**Comment by cacqw7** [comment]
:+1:

**Comment by fanquake** [comment]
Thanks :+1:

**Pull request merged**